### PR TITLE
Functions for combining gnss and weather model delays

### PR DESCRIPTION
**Pull request by jlmaurer**
## Description

Adding in functions for combining GNSS ZTD and weather model delay estimates

## How Has This Been Tested?

Has not been tested

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have added an explanation of what your changes do and why you'd like us to include them.
- [ ] I have written new tests for your core changes, as applicable.
- [ ] I have successfully ran tests with your changes locally.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: tools/RAiDER/gnss/processDelayFiles.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    '''
+    Merge a combined RAiDER delays file with a GPS ZTD delay file
+    '''
+    dfr = pd.read_csv(raiderFile)
+    dfz = readZTDFile(ztdFile)
+    dfc = dfr.merge(dfz[['ID', 'Datetime', 'ZTD']], how='inner', left_on=['Datetime', 'ID'], right_on=['Datetime', 'ID'], sort=True)
+    if outName is None:
+        return dfc
+    else:
+        dfc.to_csv(outName, index=False)
```
Comment (dbekaert) [inline tools/RAiDER/gnss/processDelayFiles.py:61]: Does this make the assumption it ingest raider computed delays where at the GNSS locations of matching time and station?

What happens if the GNSS file has every 5 min and raider is for fixed UTC.
I understand that the filename written file will then we still every 5 min, but there will be a lot of blanks.

File: tools/RAiDER/gnss/processDelayFiles.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import datetime
+import glob
+import os
+
+import pandas as pd
```
Comment (dbekaert) [inline tools/RAiDER/gnss/processDelayFiles.py:5]: @jlmaurer  Could you ad in a brief run-example. Or add a a parser __main__ for now.

File: tools/RAiDER/gnss/processDelayFiles.py (hunk 1 above)
Comment (jlmaurer) [reply tools/RAiDER/gnss/processDelayFiles.py:61]: Right now I  merge only on the intersection, so if dates are entirely missing in either file they are dropped. Ie inner join.

File: tools/RAiDER/gnss/processDelayFiles.py (hunk 2 above)
Comment (jlmaurer) [reply tools/RAiDER/gnss/processDelayFiles.py:5]: Yes, I'll add that tonight after I get out of service!

Comment (dbekaert) [reply tools/RAiDER/gnss/processDelayFiles.py:5]: Ok do you want me to wait to merge it to have the example added?

Comment (jlmaurer) [reply tools/RAiDER/gnss/processDelayFiles.py:5]: Sure that's fine, either way is fine with me.

Comment (jlmaurer) [reply tools/RAiDER/gnss/processDelayFiles.py:5]: I'll add an rough example to the comments

**Comment by jlmaurer** [comment]
Ipython example to get started: 

```
from processDelayFiles import *

loc = '.'
raider_file = 'raider_delays.csv'
ztd_file = 'UNRcombined_gnss.csv'

combineDelayFiles(raider_file, loc=loc)

 mergeDelayFiles(raider_file, ztd_file, 'Combines_delays.csv')
```

**Review by dbekaert — APPROVED** [review]
LGTM @jlmaurer  thanks

**Pull request merged**